openslides/users/static/js/users/site.js

```diff
@@ -686,10 +688,27 @@ angular.module('OpenSlidesApp.users.site', [
 
         // Export as PDF
         $scope.pdfExportUserList = function () {
-            var filename = gettextCatalog.getString("List of participants")+".pdf";
-            var userListContentProvider = UserListContentProvider.createInstance($scope.usersFiltered, $scope.groups);
-            var documentProvider = PdfMakeDocumentProvider.createInstance(userListContentProvider);
-            PdfCreate.download(documentProvider.getDocument(), filename);
+            var imageMap = {};
+            var imagePromises = [];
+            var imageSources = [];
+
+            imageSources.push(Config.get('logo_pdf_header').value.path);
+            imageSources.push(Config.get('logo_pdf_footer').value.path);
+
+            imagePromises = _.map(imageSources, function (image_source) {
```

Comment:
Yes, I agree to split this between documentProvider/contentProvider, but for the first cleanup, this is too much ;)